ipv-core/src/main/java/com/ibm/whc/deid/providers/masking/ConditionalMaskingProvider.java

```diff
@@ -135,15 +149,57 @@ private boolean evaluateConditionTypeString(String identifier,
             return true;
           }
           break;
+        case ANY_OF:
+          if (conditionValueSet == null) {
```

Comment:
I am not sure why we need to check for nulls here.  Isn't conditionValueSet/conditionValueSetLowerCase always null at this point?

Reply:
The check is in a loop, so it will be null the first time, but it might not be on a later iteration.  It might be more complicated than necessary.